libraries/core_libs/network/src/tarcap/taraxa_capability.cpp

```diff
@@ -40,7 +40,7 @@ TaraxaCapability::TaraxaCapability(std::weak_ptr<dev::p2p::Host> host, const dev
       node_stats_(nullptr),
       packets_handlers_(std::make_shared<PacketsHandler>()),
       thread_pool_(std::make_shared<TarcapThreadPool>(conf.network_packets_processing_threads, node_addr)),
-      periodic_events_tp_(1, false) {
+      periodic_events_tp_(std::make_shared<util::ThreadPool>(1, false)) {
```

Comment:
we should probably have more than 1 thread here as we are starting to use it for many things ?

Reply:
Increase threads to 2 and put that in a constant